README.md

```diff
@@ -1,10 +1,11 @@
-
 <h1 align="center">
 <img src="https://github.com/pyfar/gallery/raw/main/docs/resources/logos/pyfar_logos_fixed_size_pyrato.png" width="300">
 </h1><br>
 
+
+
 [![PyPI version](https://badge.fury.io/py/pyrato.svg)](https://badge.fury.io/py/pyrato)
-[![Documentation Status](https://readthedocs.org/projects/pyrato/badge/?version=stable)](https://pyrato.readthedocs.io/en/stable/?badge=stable)
+[![Documentation Status](https://readthedocs.org/projects/pyrato/badge/?version=latest)](https://pyrato.readthedocs.io/en/latest/?badge=latest)
```

Comment:
This should link to stable, which is the most recent version tag, not latest, which is the latest git commit on the default branch.